packages/indexer-agent/src/network.ts

```diff
@@ -1065,6 +1096,78 @@ export class Network {
     }
   }
 
+  async claimMany(allocations: Allocation[]): Promise<boolean> {
+    const logger = this.logger.child({
```

Comment:
I would move the allocations property out of this logger instantiation and instead include it in the `Claim tokens from the rebate pool for many allocations` log message. I think it'll be unnecessarily verbose to include entire allocations list in the subsequent log lines that are for specific allocations.